src/LabelComponent.jsx

```diff
@@ -0,0 +1,158 @@
+import { useState, useEffect } from "react";
+import "./LabelComponent.css";
+
+const LabelComponent = () => {
+  const [inputValue, setInputValue] = useState("");
+  const [labelsData, setLabelsData] = useState([]);
+  const [filteredSuggestions, setFilteredSuggestions] = useState([]);
+  const [selectedLabels, setSelectedLabels] = useState([null]);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch(
+          `http://localhost:3000/api/labels?query=${inputValue}`
```

Comment:
You could use an .env variable to define the backend server and use that in the code